tests/test_daemon_query.py

```
@@ -0,0 +1,284 @@
+        from tldr.cli import main
+
+        project_path = Path("/fake/project").resolve()
+        expected_response = {"status": "ok", "message": "pong"}
+
+        mock_query_func = MagicMock(return_value=expected_response)
+
+        with patch("tldr.daemon.query_daemon", mock_query_func):
+            # Simulate CLI call: tldr daemon query ping
+            test_args = ["daemon", "query", "ping", "--project", str(project_path)]
+
+            with patch("sys.argv", ["tldr"] + test_args):
```

Ruff (0.14.13)

Consider `["tldr", *test_args]` instead of concatenation

Replace with `["tldr", *test_args]`

(RUF005)